ios/App/App/AppDelegate.swift

```diff
@@ -11,6 +13,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+		// if #available(iOS 10, *) {
+		// 	let center = UNUserNotificationCenter.current()
+		// 	center.delegate = self as? UNUserNotificationCenterDelegate
+		// 	var options: UNAuthorizationOptions = [.alert, .sound, .badge]
+		// 	if #available(iOS 12.0, *) {
+		// 		options = UNAuthorizationOptions(rawValue: options.rawValue | UNAuthorizationOptions.provisional.rawValue)
+		// 	}
+		// 	center.requestAuthorization(options: options) { (granted, error) in
+		// 		Appboy.sharedInstance()?.pushAuthorization(fromUserNotificationCenter: granted)
+		// 	}
+		// 	UIApplication.shared.registerForRemoteNotifications()
+		// } else {
+		// 	let types : UIUserNotificationType = [.alert, .badge, .sound]
+		// 	let setting : UIUserNotificationSettings = UIUserNotificationSettings(types:types, categories:nil)
+		// 	UIApplication.shared.registerUserNotificationSettings(setting)
+		// 	UIApplication.shared.registerForRemoteNotifications()
+		// }
```

Comment:
braze push not working?

Reply:
deleted this file

Reply:
It was so I could compare when merging

Reply:
Yeah unfortunately, I wanted to keep the commented out code in there for future reference

Reply:
I can continue messing with it, but will likely need some assistance from the Braze team, especially since I needed to modify their example code to even get the request for push notifications to trigger.

I can continue messing with it locally, but want to at least get this code in the codebase